doc/user-guide/dask.rst

```diff
@@ -533,61 +436,50 @@ Notice that the 0-shaped sizes were not printed to screen. Since ``template`` ha
    As :py:func:`map_blocks` loads each block into memory, reduce as much as possible objects consumed by user functions.
    For example, drop useless variables before calling ``func`` with :py:func:`map_blocks`.
 
+Deploying Dask
+--------------
 
+By default, Dask distributes work across multiple cores on a single machine, which lets you process as much data as you can download and fit onto your hard drive. However, this has two limitations:
 
-Chunking and performance
-------------------------
+- You are limited by the size of your hard drive
+- Downloading data can be slow and expensive
 
-The ``chunks`` parameter has critical performance implications when using Dask
-arrays. If your chunks are too small, queueing up operations will be extremely
-slow, because Dask will translate each operation into a huge number of
-operations mapped across chunks. Computation on Dask arrays with small chunks
-can also be slow, because each operation on a chunk has some fixed overhead from
-the Python interpreter and the Dask task executor.
+Instead, it can be faster and cheaper to run your computations close to where your data is stored, distributed across many machines on a Dask cluster. Often, this means deploying Dask on HPC clusters or on the cloud. See the `Dask documentation <https://docs.dask.org/en/stable/deploying.html?utm_source=xarray-docs>`__ for more details.
 
-Conversely, if your chunks are too big, some of your computation may be wasted,
-because Dask only computes results one chunk at a time.
 
-A good rule of thumb is to create arrays with a minimum chunksize of at least
-one million elements (e.g., a 1000x1000 matrix). With large arrays (10+ GB), the
-cost of queueing up Dask operations can be noticeable, and you may need even
-larger chunksizes.
+Best Practices
+--------------
 
-.. tip::
+Dask is pretty easy to use but there are some gotchas, many of which are under active development. Here are some tips we have found through experience. We also recommend checking out the `Dask best practices <https://docs.dask.org/en/stable/array-best-practices.html?utm_source=xarray-docs>`_.
 
-   Check out the `dask documentation on chunks <https://docs.dask.org/en/latest/array-chunks.html>`_.
+.. code-block:: python
 
-.. tip::
+    from flox.xarray import xarray_reduce
+    import xarray
```

Comment:
This example seem unconnected to the surrounding text?

Reply:
Yeah, I was wondering if this made sense.

I was trying to have the code snippet be a demonstrative, simple example of putting some of the optimization tips together. I swapped the order and added a sentence of explanation. Maybe that's better? Open to other ideas!